index.js

```diff
@@ -1,42 +1,67 @@
-	const baseStatsElement = document.getElementById("base__stats");
-
-	loadButton.addEventListener("click", async () => {
-		const pokemonNumber = pokemonNumberInput.value;
-		if (!pokemonNumber) return;
-
-
-		// Як я зрозумів, можна ще використати таку структуру
-		// fetch(`https://pokeapi.co/api/v2/pokemon/${pokemonNumber}/`)
-		// .then((response) => response.json())
-		// .then((data) => 	
-		// але я не розібрався, як то записати правильно
-
-		try {
-			const response = await fetch(`https://pokeapi.co/api/v2/pokemon/${pokemonNumber}/`);
-			const data = await response.json();
-			pokemonImage.src = data.sprites.front_default;
-			pokemonImage.style.display = "block";
-
-			pokemonNameElement.textContent = `Ім'я: ${data.name}`;
-			pokemonHeightElement.textContent = `Зріст: ${data.height} см`;
-			pokemonSizeElement.textContent = `Вага: ${data.weight * 100} грам`;
-
-			baseStatsElement.innerHTML = "";
-			data.stats.forEach((stat) => {
-				const listItem = document.createElement("li");
-				listItem.textContent = `${stat.stat.name}: ${stat.base_stat}`;
-				baseStatsElement.appendChild(listItem);
+
+
+
+	const todoForm = document.querySelector("#todoForm");
+	const todoInput = document.querySelector("#todoInput");
+	const deadlineDate = document.querySelector("#deadLineDate");
+	const todoList = document.querySelector("#todoList");
+
+
+
+
+	todoForm.addEventListener("submit", function (e) {
+		e.preventDefault();
+
+
+		const todoText = todoInput.value;
+		const toDate = deadlineDate.value;
+
+
+		const todoItem = createTodoItem(todoText, toDate);
+		todoList.appendChild(todoItem);
+
+		todoInput.value = "";
+		deadlineDate.value = "";
+
+		function createTodoItem(text, toDate) {
+			const todoItem = document.createElement("li");
+			todoItem.classList.add("todo-item");
+
+			const checkbox = document.createElement("input");
+			checkbox.type = "checkbox";
+			checkbox.classList.add("checkbox");
+
+			const todoText = document.createElement("span");
+			todoText.textContent = text;
+
+			const toDateElement = document.createElement("span");
+			toDateElement.textContent = toDate ? `Last date to do: ${toDate}` : "";
+
+			const removeButton = document.createElement("button");
+			removeButton.classList.add("remove-button");
+			removeButton.textContent = "Remove";
+			removeButton.addEventListener("click", function () {
+				todoList.removeChild(todoItem);
 			});
-		} catch (error) {
-			console.error(error);
-			alert("Такого покемона, нажаль, не існує :)");
-		}
+
+			todoItem.appendChild(checkbox);
+			todoItem.appendChild(todoText);
+			todoItem.appendChild(toDateElement);
+			todoItem.appendChild(removeButton);
+
+			checkbox.addEventListener("change", function () {
+				if (checkbox.checked) {
+					todoText.style.textDecoration = "line-through";
+					toDateElement.style.textDecoration = "line-through";
+				} else {
+					todoText.style.textDecoration = "none";
+					toDateElement.style.textDecoration = "none";
+				}
+			});
```

Comment:
Тут треба повернути з функції створений елемент. Бо нічого не повертається, і вище (рядок 21-22) при спробі додати створений елемент все ламається (якраз тому що нічого не повертається з функції)

```suggestion
			});
	return todoItem
```